libcxx/test/std/iterators/iterator.requirements/indirectcallable.traits/indirect.value.t.compile.pass.cpp

```diff
@@ -0,0 +1,75 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Let's make this a `.pass.cpp` test to actually run it as well.

Reply:
Done. Although it's made to run for constant evaluation only since C++23 due to the use of `unique_ptr`.